src/test/java/com/google/api/codegen/testdata/java_page_streaming_response_library.baseline

```diff
@@ -45,9 +60,25 @@ public class FindRelatedBooksPagedResponse extends PagedListResponse<FindRelated
  */
 package com.google.gcloud.pubsub.spi;
 
+import com.google.api.gax.grpc.CallContext;
+import com.google.api.gax.grpc.FutureCallable;
+import com.google.api.gax.grpc.PageStreamingDescriptor;
+import com.google.api.gax.grpc.PagedListResponseImpl;
+import com.google.example.library.v1.Book;
+import com.google.example.library.v1.ListBooksRequest;
+import com.google.example.library.v1.ListBooksResponse;
 
 @javax.annotation.Generated("by GAPIC")
-public class ListBooksPagedResponse extends PagedListResponse<ListBooksRequest, ListBooksResponse, Book> {
+public class ListBooksPagedResponse extends PagedListResponseImpl<ListBooksRequest, ListBooksResponse, Book> {
+
+  public ListBooksPagedResponse(
+      ListBooksRequest request,
+      FutureCallable<ListBooksRequest, ListBooksResponse> callable,
```

Comment:
I would prefer to have this take a UnaryApiCallable (and consequently, PagedListResponseImpl also). You can easily wrap in the place where the FutureCallable is created by doing UnaryApiCallable.create(futureCallable).